docs/overrides/home.html

```diff
@@ -0,0 +1,188 @@
+{% extends "main.html" %}
```

Comment:
Still not fan of centering, explained here 
https://uxmovement.com/content/why-you-should-never-center-align-paragraph-text/
https://thewebsitearchitect.com/does-center-aligned-text-matter-for-accessibility/

Reply:
You are absolutely right. It's much less readable and accessible. But visually it becomes different.
I would rather readjust and have it validated by a feedback from the team. I note it and make an issue in this sense.